ServerCore/Helpers/TeamHelper.cs

```diff
@@ -39,5 +55,54 @@ public static async Task DeleteTeamAsync(PuzzleServerContext context, Team team)
 
             await context.SaveChangesAsync();
         }
+
+        public static async Task<Tuple<bool, string>> AddMemberAsync(PuzzleServerContext context, Event Event, EventRole EventRole, int teamId, int userId, int applicationId)
```

Comment:
Could you add a function comment block describing the parameters and return values?

Reply:
Added :)